src/data/unitOptions.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
obv don't have context on the game but is `unitOptions` the right name? That sounds more like customization of individual units. `unitDefinitions` or `armyDefinitions` perhaps?

Reply:
Good points Major data update coming soon